keras_cv/models/generative/stable_diffusion/clip_tokenizer.py

```diff
@@ -101,6 +101,14 @@ def __init__(self, bpe_path=None):
             re.IGNORECASE,
         )
 
+    @property
+    def end_of_text(self):
+        return self.encoder["<|endoftext|>"]
+
+    @property
+    def start_of_text(self):
```

Comment:
This token is also used as a magic number in `stable_diffusion.py` as:
```
phrase = inputs + [49407] * (MAX_PROMPT_LENGTH - len(inputs))
```

Maybe we could also replace it there with self.tokenizer.end_of_text?

(It might also be nice to add a comment to `constants.py` explaining this for the `UNCONDITIONAL_TOKENS`

Reply:
Will do in a separate branch when I update `add_tokens()`.